packages/python-packages/apiview-gpt/app.py

```diff
@@ -0,0 +1,15 @@
+from flask import Flask, request, jsonify
+from src import review_python
+
+app = Flask(__name__)
+
+@app.route('/')
+def index():
+    return 'Hello, World!'
```

Comment:
Is this really necessary?

Reply:
I use it to detect if service is online.

I will remove it.